tests/php/Security/SecurityTest.php

```diff
@@ -214,11 +214,12 @@ public function testLogInAsSomeoneElse()
         );
 
         /* We get a good response */
-        $this->assertEquals($response->getStatusCode(), 200, 'We have a 200 OK response');
-        $this->assertNotNull($response->getBody(), 'There is body content on the page');
+        $this->assertEquals($response->getStatusCode(), 302, 'We have a redirection response');
 
         /* Log the user out */
         Security::setCurrentUser(null);
+
+        $this->autoFollowRedirection = true;
```

Comment:
You're right - I've followed a pattern from another test case in the same class. It turned out to be unnecessary because the line in the middle of this test case was removed, making the "outer" ones obsolete. I've removed them now.